e2e_tests/run_fork.py

```diff
@@ -0,0 +1 @@
+print("forked")
```

Comment:
you should probably put it into some subdir such as `e2e_tests/tests/fixtures/something`